adapter/stdioLogger/stdioLogger.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // Package stdioLogger provides an implementation of the mixer logger aspect
-// that writes logs (serialized as JSON) to a standard expectedStream (stdout | stderr).
+// that writes logs (serialized as JSON) to a standard stream (stdout | stderr).
```

Comment:
This is about the format of the payload string itself.  It is the difference between:

`"textPayload":"{\"val\":\"42\", \"obj\":{\"val\":\"false\"}}"}`
and
`"structPayload":{"obj":{"val":"false"},"val":"42"}`

in the output log.

However, I think the confusion alone means that this should be aspect configuration and NOT adapter config and that Entry itself should have different values. I'm making those changes now.